recipes/libbluetooth-headers/meta.yaml

```
@@ -0,0 +1,28 @@
+{% set name = "libbluetooth-headers" %}
+{% set version = "5.56" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.kernel.org/pub/linux/bluetooth/bluez-{{ version }}.tar.xz
+  sha256: 59c4dba9fc8aae2a6a5f8f12f19bc1b0c2dc27355c7ca3123eed3fe6bd7d0b9d
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/bluetooth/bluetooth.h
+
+about:
+  home: http://www.bluez.org/
+  summary: "libbluetooth header files"
+  license: LGPL-2.1-only
+  license_file: COPYING.LIB
+
+extra:
+  recipe-maintainers:
+    - johnwason
```

```suggestion
extra:
  feedstock-name: bluez
  recipe-maintainers:
    - johnwason
```
The project from which you are taking the headers is called bluez. Let's name the feedstock with the project name, so that additional outputs from this project may be added in the future.